src/utils/render-fragments.js

```diff
@@ -385,28 +442,43 @@ export function getInputRulesForRule(
 			<div className="meta">
 				<span className="heading">Input Rules</span>
 				<ul>
-				{inputRules.map(inputRuleId => {
-					const atomicRule = allRules.find(
-						rule => rule.node.frontmatter.id === inputRuleId
-					)
-					const aHref = stripBasePath
-						? atomicRule.node.fields.slug.replace('rules/', '')
-						: atomicRule.node.fields.slug
-					const name = atomicRule.node.frontmatter.name
-					return (
-						<li key={inputRuleId}>
-							<a className="sc-item block" href={aHref}>
-								{name}
-							</a>
-						</li>
-					)
-				})}
+					{inputRules.map(inputRuleId => {
```

Comment:
Please try to keep your logic code a little separated from your UI code. It's hard to read.